packages/experimental/Shimmer/src/Shimmer.types.ts

```diff
@@ -1,114 +1,156 @@
-  xPos?: number;
+  cy: number;
+}
+
+/**
+ * Properties necessary to define a rectangular Shimmer element.
+ */
+export interface ShimmerRectElement {
   /**
-   * y-coordinate
-   * @defaultValue '0'
+   * Shimmer element type.
    */
-  yPos?: number;
-}
+  type: 'rect';
 
-export interface ShimmerTokens {
   /**
-   * Specifies the distance traveled from starting position.
-   * By default, the animation will animate from 0 to 30 over the course of 7000ms, since speed = distance / time, so
-   * speed ≈ 0.004 (higher the speed, faster it moves).
-   * @defaultValue '30'
+   * Width of the rect.
+   * @defaultValue 100%
    */
-  toValue?: number;
+  width?: number;
+
   /**
-   * Specifies the time required to travel a given distance in milliseconds
-   *
-   * @defaultValue '7000'
+   * Height of the rect.
+   * @defaultValue 16
    */
-  duration?: number;
+  height?: number;
+
   /**
-   * Specifies the animation delay time in milliseconds
-   * @defaultValue '0'
+   * Border radius for the x-axis of a rounded rect.
+   * @defaultValue 0
    */
-  delay?: number;
+  borderRadiusX?: number;
+
   /**
-   * Gradient tint color
-   * @defaultValue 'white'
+   * Border radius for the y-axis of a rounded rect.
+   * @defaultValue 0
    */
-  gradientTintColor?: ColorValue;
+  borderRadiusY?: number;
+
   /**
-   * Shimmer element tint color, no-op for image based shimmer
-   * @defaultValue 'E1E1E1' for light mode, '404040' for dark mode
+   * Note: x and y should be optional properties [or removed], with relative positioning being the default [or only] positioning mechanism.
+   * Note: The behavior of element intersections is undefined.
    */
-  shimmerTintColor?: ColorValue;
+
   /**
-   * Width of the shimmer view
-   * @defaultValue '200'
+   * The x-axis position of the rect element's top-left corner in the Shimmer relative to the origin [top-left, (0,0)].
+   * @defaultValue 0
    */
-  width?: number;
+  x: number;
+
   /**
-   * Height of the shimmer view
-   * @defaultValue '100'
+   * The y-axis position of the rect element's top-left corner in the Shimmer relative to the origin [top-left, (0,0)].
+   * @defaultValue 0
    */
-  height?: number;
+  y: number;
+}
+
+export interface ShimmerTokens extends IBackgroundColorTokens {
   /**
-   * Specifies the gradient angle, value should be anywhere between -1 and 1
-   * -1 to 1 maps to 90 degrees horizontally to ≈ 120 degrees
-   * @defaultValue '0'
+   * Specifies the Shimmer effect angle in degrees (produced by a gradient)
+   * @defaultValue 0
    */
   angle?: number;
+
+  /**
+   * Specifies the animation delay time in milliseconds
+   * @defaultValue 0
+   */
+  delay?: number;
+
+  /**
+   * Specifies the time required to traverse the control in milliseconds
+   *
+   * @defaultValue 2000
+   */
+  duration?: number;
+
   /**
-   * Specifies the gradient opacity, value should be less than 1
-   * @defaultValue '.7'
+   * Specifies the gradient opacity.
+   * @defaultValue 1
```

Comment:
Is this meant to change from 0.7?

Reply:
I did mean to change that, I haven't seen what the value is in the default shimmer being translucent -- or icon being a top line slot/prop (also removed) for that matter as a specialized use case, which was my understanding of the built-in use of the translucency.  It sounds more likely to cause confusion w/ a background color bleeding through, IMO.

I'm open to thoughts if there's more background; fwiw it's not a part of the current web shimmer or northstar skeleton interfaces